docs/risk/risk-faq.md

```diff
@@ -14,13 +14,13 @@ Moreover, another class of protocols built on top of EigenLayer is called liquid
 
 Yes, EigenLayer does reuse the same economic security by enabling different AVSs to share a common economic security base. This concept is known as 'pooled security.' Similar to how various dApps share Ethereum's economic security, EigenLayer allows different decentralized PoS networks share the same economic security base.
 
-Pooling allows various AVSs to contribute to a larger, shared economic security base, enhancing economic safety. This is because the cost to compromise any individual AVS increases significantly. For instance, consider 100 AVSs each with \$1B in economic security versus 100 AVSs sharing \$100B in economic security. The cost to attack any single AVS in the pooled system has effectively multiplied by 100.
+Pooling allows various AVSs to contribute to a larger, shared economic security base, enhancing economic safety. This is because the cost to compromise any individual AVS increases significantly. For instance, consider the scenario where 100 AVSs each with \$1B in economic security versus 100 AVSs sharing \$100B in economic security. The cost to attack any single AVS in the pooled system has effectively multiplied by 100.
 
-In addition to pooled security, EigenLayer will let each AVS acquire additional attributable security. Unlike pooled security, attributable security is specific to each AVS and will only be slashable by one AVS. This arrangement can provide game-theoretic guarantees for AVS customers. 
+In addition to pooled security, EigenLayer will let each AVS acquire additional attributable security. Unlike pooled security, attributable security is specific to each AVS and will only be slashable by one AVS. Under this design, the end goal is to provide unconditional economic safety (as defined in the StakeSure paper) for AVS end consumers. Using a bridge AVS as an example, it has a shared security base of \$10B and an attributable portion of \$10B. If the shared security part is slashed, the AVS can assure the bridge customer that up to \$10B can still be securing the bridge through the attributable part. 
```

Comment:
Please italicize _unconditional economic safety_ (or wrap in quotes) so it's more clear it's an outside concept

And please link the StakeSure paper

Reply:
added the change here: https://github.com/Layr-Labs/eigenlayer-docs/pull/62/commits/6604b61678ea5a1f7f4eeec82f36a1560473127f